saleor/account/i18n.py

```diff
@@ -185,12 +189,24 @@ def validate_address(self, data):
                 del data["sorting_code"]
         except i18naddress.InvalidAddressError as exc:
             self.add_field_errors(exc.errors)
+            self.log_errors()
         return data
 
     def clean(self):
         data = super().clean()
         return self.validate_address(data)
 
+    def log_errors(self):
+        errors = self.errors
+        fields = {}
+        for field, _ in errors.items():
+            fields[field] = (
+                self.data.get(field) if field in ADDRESS_FIELDS_TO_LOG else "invalid"
+            )
+        fields["skip_validation"] = self.data.get("skip_validation")
```

Comment:
Can't you just add `skip_validation` and `country` to `ADDRESS_FIELDS_TO_LOG` ?

Reply:
No, because `skip_validation` and `country` fields will not contain any error. I want to log it always when at least single error occur